### ISSUE TITLE
Promotion: Free fanny-pack on orders over $100 [has estimate]

### ISSUE THREAD

**Issue by justinbmeyer** [bitovi/continous-exploration#2]
### Customer Requirements

[See wireframes for promotions here](https://docs.google.com/presentation/d/1bFMezSqqTuMyzOGR0P8H-neSNuwZEsmxgYAKG9mcuq4/edit#slide=id.p)

__Happy Path__ 

- Customer is able to see the fanny-pack promotion on the homepage
- Customer clicks the fanny-pack discount and is prompted to add the fanny-pack to the cart
- Customer adds the fanny-pack and promotion to the cart
- Customer sees a warning about needing $100 to use the promotion.
- Customer adds $100 of value to their cart.
- Customer checks out
- Customer sees the promotion on their receipt

__Refund Case__

- Blah Blah Blah

__Customer Removes Promotion Case__

- Blah Blah Blah

__Customer Removes Fanny Pack Case__

- Blah Blah Blah

### Administration User Requirements

- Administrator is able to create a fanny-pack promotion
- Administrator is able to disable the fanny-pack promotion

## Out of scope

- We will only support free-items given some minimal order amout.
- Promotions not exist for some "timeframe", a administrator will need to enable and disable.
- Promotion will be available once per order. It will not be limited to once-per-customer.

## Epics 📝

- Promotion - Provide a list of promotions for the user
  - Promotion - Receive a list of promotions 
    - Administration - Build and publish discounts 
- Cart - Enable a promotion to be added to the cart 
  - Cart - price the promotion 
  - Promotions - Verify if a promotion is available
- Order - Place an order with a promotion
  - Promotions - Consume the promotion
     - Order - Send the promotion to the store

**Comment by justinbmeyer** [comment]
## What is the `has estimate` list?

This is a list of initiatives whose:

- high-level requirements have been scoped
- work has been roughly estimated and planned

but the initiatives have _not_:

- had their value proven or estimated

## How to get an initiative to `has estimate`?

__First__, break down the work across by user requirement.  Nest stories/epics that must be done prior to completing their parent story/epic.

For example, the following must be built to complete the "Customer is able to see the fanny-pack promotion on the homepage" user requirement:

```
- Promotion - Provide a list of promotions for the user
  - Promotion - Receive a list of promotions
   - Administration - Build and publish discounts
```

__Next__, create these stories/epics and ask the engineering teams to give their estimate of the work.  

> PRO TIP: Estimates should include some level of confidence.  

__Finally__, break down the work visually in roadmapping software. This can often help you improve the timing and de-risk development. 

![image](https://user-images.githubusercontent.com/78602/107165470-a464e600-6978-11eb-83e4-4397bfd51523.png)


## How to sort the `has estimate` list?

This should be sorted by each initiative's expected value.
